### ISSUE TITLE
ingest and catalogue Community OSCAR

### ISSUE THREAD

**Issue by jsaizant** [OpenEuroLLM/training-data-catalogue#17]
A multilingual, CommonCrawl-based corpus spanning 45 dumps from 2014 to 2024. It was processed based on quality heuristic warnings, harmful perplexity, FastText language identification and a URL-based blocklist, as well as TLSH deduplication.

Repo: https://huggingface.co/datasets/oscar-corpus/community-oscar
Paper: https://aclanthology.org/2024.mrl-1.19.pdf

**Comment by jindrahelcl** [comment]
The paper mentions the following

> The raw Community OSCAR
data should be processed further before being used
for LLM training. To begin with, different crawls
contain large amounts of duplicate documents. Additionally, the raw data from CC consists of different quality levels concerning syntactical and grammatical correctness, factual accuracy, quality of
HTML parsing, unsafe content, etc.

which probably means we should not be including this in the catalogue which is aimed at providing data for LLM training

**Comment by jindrahelcl** [comment]
They refer to https://huggingface.co/datasets/occiglot/occiglot-fineweb-v0.5 as a preliminary filtered version of the raw corpus

**Comment by oepen** [comment]
in the end, we decided against inclusion in the catalogue.  community oscar, in its current form, is more of a data pool then a prefiltered training dataset.
